Program.cs

```diff
@@ -26,6 +27,7 @@ public static async Task Main(string[] args)
         collection.AddAssignableTo<IRconCommand>();
         collection.AddAssignableTo<IAuthenticationTask>();
         collection.AddAssignableTo<IUserDataLoadingTask>();
+        collection.AddAssignableTo<IProfileManager>();

```

Comment:
Not needed. Services are automatically matched based on the interface and implementation name. Rule is `**I**Service` -> `Service`